server/src/BitBakeProjectScanner.ts

```diff
@@ -99,13 +99,13 @@ export class BitBakeProjectScanner {
       void _connection?.sendNotification('bitbake/startScan')
 
       try {
-        this.parseAllRecipes()
         this.scanAvailableLayers()
         this.scanForClasses()
         this.scanForIncludeFiles()
         this.scanForRecipes()
         this.scanRecipesAppends()
         this.scanOverrides()
+        this.parseAllRecipes()
 
         logger.info('scan ready')
```

Comment:
I think here it actually means `scan finished`.

Reply:
This code was not added by this PR (I think it's from Eugen's version), but yes, `scan finished` would be more accurate as it can also have failed.